models/transfer_check.go

```diff
@@ -290,24 +296,26 @@ func (t TransferDataCreateBody) FormatAndValidate() (TransferData, error) {
 
 	out.Currency = strings.ToUpper(t.Currency)
 	if !slices.Contains(pure_utils.CurrencyCodes, t.Currency) {
-		errs["currency"] = fmt.Sprintf("currency %s is not valid", t.Currency)
+		errs["currency"] = fmt.Sprintf("currency '%s' is not valid", t.Currency)
 	}
 
 	if t.SenderIP != "" {
 		ip, err := netip.ParseAddr(t.SenderIP)
 		if err != nil {
-			errs["sender_ip"] = fmt.Sprintf("sender_ip %s is not a valid IP address", t.SenderIP)
+			errs["sender_ip"] = fmt.Sprintf("sender_ip '%s' is not a valid IP address", t.SenderIP)
 		} else {
 			out.SenderIP = ip
 		}
 	}
 
-	if !slices.Contains(TransferStatuses, t.Status) {
-		errs["status"] = fmt.Sprintf("status %s is not valid", t.Status)
+	if t.Status == "" {
```

Comment:
IIUC, it's optionnal on the sent payload but we default to Neutral (so not optionnal on received body from the DB in the end).

It does matter to me since we use a radio button on the frontend

Reply:
Yep, sorry the title was unclear.
We were always supposed to default to neutral, but I broke that when adding "input enum validation" on it.